api/resources_portal/views/organization_invitations.py

```diff
@@ -28,16 +28,28 @@ class OrganizationInvitationViewSet(viewsets.ModelViewSet):
     serializer_class = OrganizationInvitationSerializer
 
     def update_organizations(self, new_status, invitation):
-        old_status = invitation.status
-        if old_status == "PENDING" and new_status == "ACCEPTED":
+        if new_status == "ACCEPTED":
             invitation.organization.members.add(invitation.requester)
-            # TODO: Add notification of acceptance
-        elif old_status == "PENDING" and new_status == "REJECTED":
-            pass
-            # TODO: Add notification of rejection
-        elif old_status == "PENDING" and new_status == "INVALID":
-            pass
-            # TODO: Add notification of invalid request reciever
+            invitation.delete()
```

Comment:
Don't delete the invitations! This data won't cost much to store and may one day be useful in debugging.

Reply:
we should be doing soft delete if we need things to be deleted or look deleted